### PR TITLE
Review and update `Region` docstrings

### PR DESCRIPTION
**Pull request by swapneelap**

### CODE REVIEW

File: discretisedfield/region.py [hunk 1]
```diff
@@ -136,9 +143,9 @@ def pmin(self):
 
         Returns
         -------
-        numpy.ndarray (3,)
+        numpy.ndarray
 
-            Point with minimum coordinates :math:`(p_x^\text{min},
+            Point with minimum coordinates for tree dimensions :math:`(p_x^\text{min},
```
Comment (samjrholt) [inline discretisedfield/region.py:148]:
```suggestion
            Point with minimum coordinates. E.g. for three dimensions :math:`(p_x^\text{min},
```

Comment (samjrholt) [reply discretisedfield/region.py:148]: @swapneelap does this not give an array of `len(ndims)`?

File: discretisedfield/region.py [hunk 2]
```diff
@@ -364,7 +371,7 @@ def edges(self):
 
         Returns
         -------
-        numpy.ndarray (3,)
+        numpy.ndarray
 
              Edge lengths :math:`(l_{x}, l_{y}, l_{z})`.
```
Comment (samjrholt) [inline discretisedfield/region.py:376]:
```suggestion
             Edge lengths 3 dimensions :math:`(l_{x}, l_{y}, l_{z})`.
             Edge lengths n dimensions :math:`(l_{0}, ..., l_{n})`.
```

What do you think of something like this?
We could add the default n dimension names.

If we wanted something like this I could go through and edit all the doc strings with this example

Comment (swapneelap) [reply discretisedfield/region.py:376]: I like the the `n` dimensions idea. If you like, please go ahead and edit it everywhere.

File: discretisedfield/region.py (hunk 1 above)
Comment (swapneelap) [reply discretisedfield/region.py]: @samjrholt Sorry I did not get your point.

**Pull request merged**